src/data/package/shared_package_config.rs

```diff
@@ -48,9 +56,8 @@ impl SharedPackageConfig {
         let mut shared_package = SharedPackageConfig {
             config: package.clone(),
             restored_dependencies: shared_iter
-                // this is not needed right?
-                //.collect::<Vec<SharedPackageConfig>>()
-                //.iter()
+                .collect::<Vec<SharedPackageConfig>>()
```

Comment:
So this was needed after all ? (I don't remember why?)

Reply:
Uh probably accident, no idea